File: Nível 1/1019 - Conversão de Tempo/main.lua
numero = tonumber(io.read())

horas = math.floor(numero / 60 ^ 2)
numero = numero - horas * (60 * 60)

minutos = math.floor(numero / 60)
numero = numero - minutos * 60

segundos = numero 

print(string.format("%d:%d:%d", horas, minutos, segundos))

File: Nível 1/1019 - Conversão de Tempo/main.py
numero = int(input("A: "))

horas = numero // 60 ** 2

numero = numero - horas * 60 ** 2
minutos = numero // 60

numero = numero - minutos * 60
segundos = numero

print("{}:{}:{}".format(horas, minutos, segundos))

